README.md

```diff
@@ -9,6 +9,49 @@ The Chef Workstation documentation is stored in the `chef/chef-workstation` repo
 in the [`www`](https://github.com/chef/chef-workstation/tree/master/www) directory.
 We use [Hugo modules](https://gohugo.io/hugo-modules/) to build the [Chef Workstation documentation](https://docs.chef.io/workstation/).
 
+### Preview local changes to chef-workstation documentation from chef-web-docs
+
+Follow these steps to preview changes to the chef-workstation documentation while
+running Hugo from chef-web-docs.
+
+1. Clone `chef/chef-web-docs` and `chef/chef-workstation` into the same directory.
+
+1. Initialize a go.mod file in `chef-workstation`:
+
+   ``` bash
+   cd chef-workstation
+   hugo mod init github.com/chef/chef-workstation
+   ```
+
+1. Modify the go.mod file in `chef-web-docs`.
+
+   Add `replace github.com/chef/chef-workstation => ../chef-workstation` below the
+   `require` statement. The whole file should look like this:
+
+   ```
+   module chef/chef-web-docs
+
+   go 1.14
+
+   require github.com/chef/chef-workstation v0.0.0-20200408195456-2df22d836e01 // indirect
+
+   replace github.com/chef/chef-workstation => ../chef-workstation
+   ```
+
+1. Start the Hugo server from `chef-web-docs`:
+
+   ```
+   make serve
+   ```
+
+You can preview any changes made to the documentation in `chef-workstation` as
+they would appear on https://docs.chef.io.
+
+**Before you submit a PR**
+
+- Delete the go.mod file from `chef-workstation`.
+- Delete or comment out the `replace` directive in the chef-web-docs/go.mod file.
```

Comment:
Is there any way of deleting these automatically from PRs (with a GH action? or expeditor?) ? I feel like this step is likely to cause frustration.

Reply:
It's better now.